dll/shellext/shellbtrfs/shellext.h

```diff
@@ -222,6 +223,41 @@ typedef struct _FSCTL_SET_INTEGRITY_INFORMATION_BUFFER {
 #define FSCTL_SET_INTEGRITY_INFORMATION CTL_CODE(FILE_DEVICE_FILE_SYSTEM, 160, METHOD_BUFFERED, FILE_READ_DATA | FILE_WRITE_DATA)
 
 #endif
+#else // __REACTOS__
+#ifndef DUPLICATE_EXTENTS_DATA // NT6.3+ feature.
```

Comment:
I am not sure about NT6.3+ comment: that is according to ReactOS, though MS docs read 'Windows Server 2016'.

Reply:
`#ifndef DUPLICATE_EXTENTS_DATA` cannot be used to detect the presence of the struct `DUPLICATE_EXTENTS_DATA`, so this check makes no sense.